tests/multicluster/setup/profile_none/doc_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package setupconfig
```

Comment:
Why does multicluster need to be in a separate directory from other setup configs? Why not put this in `tests/setup/multicluster/doc_test.go` or (if we want to support multicluster with profiles other than none) in `tests/setup/profile_none_multicluster/doc_test.go`?

Reply:
As others are for single cluster, I thought of keeping this separate. Putting it under `tests/setup`.